test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -306,6 +306,11 @@ public static double randomDoubleBetween(double start, double end, boolean lower
         return result;
     }
 
+    /** A random float between 0.0 and 1.0 (inclusive). */
+    public static float randomFloatRatio() {
```

Comment:
Whats the difference of this to randomFloat()? To me it looks like that would do the trick, but might be missing something.